packages/noir-ethereum-api/src/utils/object.ts

```diff
@@ -0,0 +1,4 @@
+import { stringify } from './json-bigint.js';
+
+export const equals = (a: object | string | number, b: object | string | number): boolean =>
```

Comment:
no need to support `string` and `number`, we can use `===` instead :)

Reply:
done